### PR TITLE
New docs for Build Rules, `sl check-analysis`

### PR DESCRIPTION
**Pull request by khorne3**
* [Build Rules](https://app.gitbook.com/@shiftleft/s/shiftleft-staging/v/check-analysis/inspect/pr-workflow/build-rules)
* [`sl check-analysis` Options](https://app.gitbook.com/@shiftleft/s/shiftleft-staging/v/check-analysis/using-the-shiftleft-cli/cli-reference/check-analysis)

Existing doc to which I added references: [CLI Reference](https://app.gitbook.com/@shiftleft/s/shiftleft-staging/v/check-analysis/using-the-shiftleft-cli/cli-reference#sl-commands)

### CODE REVIEW

**Comment by khorne3** [comment]
@Preetam Can you please take a look at your convenience to see if these are on the right track? Thanks!

CC: @alokshukla1978 , @arunmecheri

File: using-cli/options/check-analysis.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+## API Token
+
+Because `sl check-analysis` calls the ShiftLeft API, you'll need to create and set a process environment variable called `SHIFTLEFT_API_TOKEN` with your API token.
+
+You can obtain your API token from the Dashboard's [Account Settings](https://www.shiftleft.io/user/profile) page under **Public API Token (Beta)**.
+
+## Usage
+
+To run: `sl check-analysis [command options] [arguments]`
```
Comment (Preetam) [inline using-cli/options/check-analysis.md]: There are no arguments so you can remove `[arguments]`.

**Comment by khorne3** [comment]
@Preetam Since this feature is in alpha (but it seems we might have need for public-facing docs already), should I mark the docs as alpha and ship (but keep the links out of the nav bar)?

**Comment by Preetam** [comment]
Yes, please!

Also one suggestion, can you change the example to

```yaml
build_rules:
  - id: SQL Injections
    severity:
      - SEVERITY_MEDIUM_IMPACT
      - SEVERITY_HIGH_IMPACT
    type:
      - SQL Injection
    owasp_category:
      - a1-injection
    threshold: 10
  - id: Low Severity
    severity:
      - SEVERITY_LOW_IMPACT
    threshold: 100
```

We want to make it clear that users can specify multiple rules in the same YAML file.

**Comment by khorne3** [comment]
@Preetam Yes, I'll update the example

**Pull request merged**